Update Chinese translations

eightfiftytwo opened the pull request:
Hi! This is my first time contributing to an FOSS project. I updated the translations for Traditional and Simplified Chinese. Please let me know if I messed something up. Thank you.

manuel-192 commented:
Thanks for the contribution!

I'm just wondering about the files you have changed. In the commits there are changes to more than one package. And for example, the WELCOME_FONT variable usage looked missing quotes.
Also, when I cloned your repository, I see changes only in package eos-translations.

So I'm a bit confused...

eightfiftytwo commented:
Originally, I changed both the translation and the Welcome app, since the Chinese text look squished and misaligned on my machine. I thought there was a problem with the default Chinese font. After fiddling around with configs I found that was not the case,  so I reverted the commits to the Welcome app. 
TLDR: I only intend to change the translations.

manuel-192 commented:
OK, thanks for the clarification.
I'll merge the translations soon.
